exercises/concept/port-palermo/port_palermo.rb

```diff
@@ -1,5 +1,5 @@
 module Port
-  # TODO: define the 'Identifier' constant
+  # TODO: define the 'IDENTIFIER' constant
```

Comment:
@meatball133 the comment earlier was specifically regarding this submitted version to the student.

Reply:
In response to [this](https://github.com/exercism/ruby/pull/1562#discussion_r1299237531) yes...